src/modules/assignment-2/questions/question9/ThemeComponent.jsx

```diff
@@ -0,0 +1,43 @@
+import { useContext } from "react";
+import { ThemeContext } from "./ThemeContext";
+import "../../index.css";
+
+const ThemeComponent = () => {
+  const theme = useContext(ThemeContext);
+
+  const themeStyle = {
+    white: {
+      height: 400,
+      width: 700,
+      fontSize: 50,
+      display: "flex",
+      justifyContent: "center",
+      alignItems: "center",
+      margin: "0 auto",
+      backgroundColor: "white",
+      color: "black",
+    },
+    black: {
+      height: 400,
+      width: 700,
+      fontSize: 50,
+      display: "flex",
+      justifyContent: "center",
+      alignItems: "center",
+      margin: "0 auto",
+      backgroundColor: "black",
+      color: "white",
+    },
+  };
+
+  return (
+    <>
+      <div style={themeStyle[theme.currentTheme]}>Hello World</div>
+      <button className="button-style" onClick={() => theme.toggleTheme()}>
```

Comment:
counter can't be less then 0 in this case 
![image](https://github.com/ayush1-successive/react-training/assets/83696224/82c1e601-209d-47f2-83e5-1641a4879555)